test/plugin/test_in_heroku_http.rb

```diff
@@ -155,24 +155,21 @@ def test_ignore_unmatched_drain_id
 
     d.run(expect_records: 0) do
       res = post(tests)
-      assert_equal "200", res.code
+      assert_equal '200', res.code
     end
 
     assert_equal(0, d.events.length)
   end
 
   def post(messages)
     # https://github.com/heroku/logplex/blob/master/doc/README.http_drains.md
-    http = Net::HTTP.new("127.0.0.1", PORT)
-    req = Net::HTTP::Post.new('/heroku', {
-      "Content-Type" => "application/logplex-1",
-      "Logplex-Msg-Count" => messages.length.to_s,
-      "Logplex-Frame-Id" => "09C557EAFCFB6CF2740EE62F62971098",
-      "Logplex-Drain-Token" => "d.fc6b856b-3332-4546-93de-7d0ee272c3bd",
-      "User-Agent" => "Logplex/v49"
-    })
+    http = Net::HTTP.new('127.0.0.1', PORT)
+    req = Net::HTTP::Post.new('/heroku', 'Content-Type' => 'application/logplex-1',
+                                         'Logplex-Msg-Count' => messages.length.to_s,
+                                         'Logplex-Frame-Id' => '09C557EAFCFB6CF2740EE62F62971098',
+                                         'Logplex-Drain-Token' => 'd.fc6b856b-3332-4546-93de-7d0ee272c3bd',
```

Comment:
You should create constants with meaningful names for the above 2 values. Also, reading the doc link above, it seems that `Drain-Token` should be unique for the log drain plugin, and `Frame-Id` should be unique per request (for later tracking/debug purposes)

Reply:
I'm not going to create constants for just the tests.

Reply:
Fair enough. I didn't notice this was in the tests.